applications/luci-app-keepalived/htdocs/luci-static/resources/view/keepalived/vrrp_instance.js

```diff
@@ -0,0 +1,286 @@
+'use strict';
+'require view';
+'require form';
+'require uci';
+'require network';
+'require tools.widgets as widgets';
+
+return view.extend({
+	load: function() {
+		return Promise.all([
+			network.getDevices(),
+			uci.load('keepalived'),
```

Comment:
Is this uci.load needed?

Reply:
Yes, If `uci.load('keepalived')` is not used in `load`, it does not list values from `uci.sections()`